dirsrvtests/tests/suites/clu/dsconf_agmt_create_test.py

```diff
@@ -0,0 +1,115 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2024 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+import os
+import logging
+import pytest
+import time
+import subprocess
+from lib389.topologies import topology_i2 as topo_i2
+from lib389.replica import ReplicationManager, Replicas
+from lib389._constants import *
+from lib389.agreement import Agreements
+from lib389.properties import *
+
+pytestmark = pytest.mark.tier1
+
+DEBUGGING = os.getenv("DEBUGGING", default=False)
+if DEBUGGING:
+    logging.getLogger(__name__).setLevel(logging.DEBUG)
+else:
+    logging.getLogger(__name__).setLevel(logging.INFO)
+log = logging.getLogger(__name__)
+
+def test_replication_agreement_creation_time(topo_i2):
```

Comment:
I wonder if  we should mark the test as flaky to avoid running it in nightly tests:
The test is based on measuring short execution time.
But the test vm sometime freezes for a few seconds, so there is a fair risk of seeing spurious failures ...

Reply:
Sounds good. I marked it flaky and I increased the time check (5 -> 10) as the issue originally was reported for 65 seconds delay so we are good.